dev-haskell/beam-migrate-cli/beam-migrate-cli-0.2.0.0_pre9999.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.1.9999
+
+CABAL_FEATURES=""
+inherit haskell-cabal git-r3
+
+DESCRIPTION="CLI tool for beam-migrate"
+HOMEPAGE="https://travis.athougies.net/projects/beam.html"
+EGIT_REPO_URI="https://github.com/tathougies/beam"
+EGIT_BRANCH="master"
+EGIT_COMMIT="66abcad4a824b9ac7f8f5c99b4c9c850c5aa082c"
```

Comment:
As you have pinned it to a particular date I suggest using specific date of that commit in ebuild name, like `beam-migrate-cli-0.2.0.0_pre20191101.ebuild`